sympy/series/limits.py

```diff
@@ -136,6 +136,11 @@ def doit(self, **hints):
         if not e.has(z):
             return e
 
+        # limit can work with binomial by converting to factorial first
+        if (e.has(binomial) and not (hints.get('sequence', True)
```

Comment:
Hi. Could you please guide how `_eval_as_tractable` should work?
I tried implementing `_eval_as_tractable` . But i was confused. For trying, I simply made it return factorial form, same as  `_eval_as_factorial`. and calling e.rewrite('tractable') in limits.py. But this was unable to evaluate the limit and returned the limit expr as such.